Application/src/controller/network/Decode.java

```diff
@@ -144,7 +143,7 @@ private String removeCharactersFromString(String string, char... characters) {
     }
 
     /**
-     * Given a string following the network protocol, will create a new diagram with the provided classes.
+     * Given a string following the controller.network protocol, will create a new diagram with the provided classes.
```

Comment:
Was this auto replaced?

Reply:
Yes ill change that back. IntelliJ did not recognize the moving around of stuff if i did not rebuild which apparently had such side effects.